ftw/iframeblock/browser/resources/iframeblock.js

```diff
@@ -1,9 +1,26 @@
-(function() {
+(function (factory) {
+  if (typeof define === 'function' && define.amd) {
+    require(['jquery', 'iframeResizer'], factory);
+  } else {
+    factory(window.jQuery, window.iFrameResize);
+  }
+}(function ($, iFrameResize) {
+  // We have to wait for the ready and then add the onload event listener directly on the
+  // iframe because the iframe does not propagate the onload back to document.
+  $(document).ready(() => {
+    $('iframe').on('load', function () {
+      var iframe = $(this);
 
-  "use strict";
```

Comment:
Could we keep `"use strict";` please? :-)

Reply:
Oops